src/Compilers/Core/Portable/Text/SourceText.cs

```diff
@@ -415,14 +415,11 @@ public virtual void Write(TextWriter writer, TextSpan span, CancellationToken ca
             }
         }
 
-        internal ImmutableArray<byte> GetChecksum(bool useDefaultEncodingIfNull = false)
+        public ImmutableArray<byte> GetChecksum()
```

Comment:
@jaredpar this is the public API change

@mattwar  are you okay with this API?